sourcegraph/sourcegraph
# Code Insights: Add timeout errors to insight card UI

Closes https://github.com/sourcegraph/sourcegraph/issues/42297

## Do not merge this PR 
This PR is part of a bigger feature insight timeout errors, this PR implements only FE change (adding timeout error icon) and adds some fake status API to the GQL schema. The BE changes should be done in this issue https://github.com/sourcegraph/sourcegraph/issues/42296. After BE issue we will be able to merge this PR

<img width="1129" alt="Screenshot 2022-10-20 at 17 42 49" src="https://user-images.githubusercontent.com/18492575/197062065-e3bb85f3-d1a5-4c48-a085-38087603e23a.png">

_Note: on the image we render all icons (next to title, and series timeout icons) this has done only in demo purpose, the current logic works different and renders either global timeout icon or icons next to the legend items_

## Test plan
- Check that you can see timeout error icon next to series legend if this series has been timeout on the backend 
- Check that you can see icon in the card (next to title) if all series have been timeout in the same time (capture group insight case)
